project_setup/scripts/skeleton

```diff
@@ -65,12 +70,20 @@ def clone_repo(parent_repo, new_repo, org, dir=None):
         f"git remote add origin git@github.com:{org}/{new_repo}.git",
         "Add a new remote origin for the this repository.",
     )
-    run(f"git tag -d $(git tag -l)", f"Delete all local git tags from {parent_repo}")
+    run("git tag -d $(git tag -l)", f"Delete all local git tags from {parent_repo}")
```

Comment:
It would be great to see GitPython used instead of these `git` commands at some point.  That's not on the CRITICAL PATH though.

Reply:
Got it!  CRITICAL PATH!